### PR TITLE
Fixing text title overflow in headers(fixes #1839)

### LINKED ISSUE #1839: Header title overflow issue
found in library,meetups and team as  well.
course-view
 Can we make separate partition for header title and action as left and right section respectively and assigned specific height -width to each?

### PR DESCRIPTION
**Pull request by BhargaviNadendla**

### CODE REVIEW

File: src/app/courses/view-courses/courses-view.scss [hunk 1]
```diff
@@ -19,4 +19,13 @@
     overflow: auto;
   }
 
+  .title {
```
[the remark this answers is not in the document]
Comment (paulbert) [reply src/app/courses/view-courses/courses-view.scss]: Yes `height`, `width`, and `font` are not needed and should be removed.  `white-space` is also already set by the Material CSS, but we should keep that to future proof this from updates on their end.

A note on fonts: `font-family` is set app wide in `styles.css`.  On a component level we should only ever change the font size, not the font itself.

File: src/app/courses/view-courses/courses-view.component.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 </mat-toolbar>
 <div class="space-container">
   <mat-toolbar class="primary-color font-size-1">
-    <span class="margin-lr-3"><h3>{{courseDetail.courseTitle}}</h3></span>
+    <span class="margin-lr-3, title">{{courseDetail.courseTitle}}</span>
```
Comment (paulbert) [inline src/app/courses/view-courses/courses-view.component.html]: This is incorrect syntax for HTML classes.  They should be separated by spaces only.

Also please keep the `h3` tags.

**Comment by lmmrssa** [comment]
@BhargaviNadendla you were able to fix this issue on last meeting but you haven't pushed any changes. Could you please update it.

**Comment by BhargaviNadendla** [comment]
@lmmrssa  On updating the `text-overflow: ellipisis`, the buttons are compressed and it looks congested on resources-view and meetups-view as below(screenshot attached). So, the buttons are padded and the width was increased.

image

File: src/app/resources/view-resources/resources-view.scss [hunk 3]
```diff
@@ -36,4 +36,10 @@
       max-height: 60vh;
     }
   }
+
+  button{
+    padding: 2px;
+    width: 20%;
```
Comment (lmmrssa) [inline src/app/resources/view-resources/resources-view.scss]: width of button should not be made fixed as button text might change

File: src/app/resources/view-resources/resources-view.scss [hunk 4]
```diff
@@ -36,4 +36,10 @@
       max-height: 60vh;
     }
   }
+
+  button{
```
Comment (lmmrssa) [inline src/app/resources/view-resources/resources-view.scss]: button styling is making resource view button to look nice. button on other pages are still breaking. Can we have the general style for title-header may be using some common style.

**Comment by paulbert** [comment]
Closing in favor of #3318

**Pull request closed without merging**